core/src/main/scala/cats/instances/either.scala

```diff
@@ -80,6 +97,20 @@ trait EitherInstances extends EitherInstances1 {
         b => s"Right(${B.show(b)})"
       )
     }
+
+  implicit def catsDataMonoidForEither[A, B](implicit B: Monoid[B]): Monoid[Either[A, B]] =
+    new Monoid[Either[A, B]] {
+      def empty: Either[A, B] = Right(B.empty)
+      def combine(x: Either[A, B], y: Either[A, B]): Either[A, B] = x combine y
+    }
+
+  implicit def catsDataSemigroupKForEither[L]: SemigroupK[Either[L, ?]] =
+    new SemigroupK[Either[L, ?]] {
+      def combineK[A](x: Either[L, A], y: Either[L, A]): Either[L, A] = x match {
```

Comment:
I'm concerned about this. Is this how Xor worked?

The standard combine is to prefer `Left`, but this `combineK` is preferring `Right` in that case. Maybe this is what you expect with `MonadCombine`, but I'm a bit concerned there is no documentation of this, and it is a bit unclear why one works in one direction, but the other the opposite.